classes/system/kibana/server/single.yml

```diff
@@ -1,11 +1,12 @@
 classes:
 - service.kibana.server.single
 parameters:
-  _param:
-    kibana_elasticsearch_host: localhost
```

Comment:
FYI, I removed this because it is already set in classes at the upper level (classes/system/stacklight/server/single.yml and classes/system/stacklight/server/cluster.yml).